lib/services/network_manager.dart

```diff
@@ -89,12 +107,12 @@ Future<void> _onConnectivityChange(
 }
 
 /// Sets the offline mode based on the current connectivity and user settings
-/// Returns true if offline mode is now enabled
+/// Always returns false for proper typing in the above Future.wait
```

Comment:
I don't really understand this

Reply:
Dart said The `final [offlineModeActive, baseUrlChanged] =` is invalid when returning a void since its not all the same datatype

Reply:
Ah right, since it's not a future